index.css

```diff
@@ -0,0 +1,361 @@
+body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Montserrat', sans-serif !important;
+    background-color: #FFF5F0;
+}
+button{
+    font-family: 'Montserrat', sans-serif !important; 
+}
+.header{
+   
+    padding-left: 10%;
+    padding-right: 5%;
+    height: 100%;
+    
+}
+.header__content{
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+}
+.header__item{
+    width: 100%;
+    margin: auto;
+}
+.header__logo{
+ font-weight: 700;
+ font-size: 24px;
+ line-height: 32px;
+}
+.header__nav-item{
+    display: inline;
+    text-align: left;
+    margin-right: 20px;
+    font-weight: 700;
+    font-size: 14px;
+    list-style: 24px;
+    color: #737373;
+}
+.header__item--middle{
+    text-align: left;
+}
+.header__item--right{
+    text-align: right;
+}
+.header__divButton{
+    display: flex;
+    float: right;
+}
+.header__button{
+    margin-right: 20px;
+}
+.header__divButton button{
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #1BA0AB;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 22px;
+    padding: 12px;
+    padding-left: 15px;
+    padding-right: 15px;
+    border: none;
+    background-color: transparent;
+    border-radius: 4px;
+}
+.header__button--active button{
+ background-color: #1BA0AB;
+ color: white ;
+ border-radius: 4px;
+ font-family: 'Montserrat', sans-serif !important;
+}
+.header__button--active button svg{
+    margin-left: 5px;
+}
+.welcome{
+    background-color: #FFF5F0;
+    padding-left: 10%;
+    padding-right: 5%;
+    margin: 0;
+    height: 70vh;
+}
+.welcome__smalltitle{
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 24px;
+    color: #1BA0AB;
+    margin-top: 150px;
+}
+.welcome__title{
+    font-weight: 700;
+    font-size: 58px;
+    line-height: 70px;
+    letter-spacing: 0.2px;
+    color: #252B42;
+}
+.welcome__desc{
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 15px;
+    letter-spacing: 0.2px;
+    color: #737373;
+    margin-bottom: 30px;
+
+}
+.welcome__footer{
+ display: flex;
+}
+.welcome__button button{
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #1BA0AB;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 22px;
+    padding: 15px;
+    padding-left: 35px;
+    padding-right: 35px;
+    border:1px solid #1BA0AB ;
+    background-color: transparent;
+    border-radius: 3px;
+    margin-right: 12px;
+}
+.welcome__button--active button{
```

Comment:
Avoid to use `button` tag instead use a class. or create a general class called `btn` that will design default button, then add defferent modifier to it